### ISSUE TITLE
Support for oVirt virtualtzation.

### ISSUE THREAD

**Issue by ThomasADavis** [samilliken/openDCIM#880]
I am working on getting oVirt to work like the ProxMox/ESX integration in the current git master, and almost done with it.

However, there is no PHP SDK for ovirt; there is however one for python, and I've created an equivalent poller script in python - is this acceptable to have just the poller script in python for the cron job?  Everything else is in PHP.

Let me know what you would like.

**Comment by samilliken** [comment]
Yes, that's fine to use Python for a support script (which is what I'd call
the poller script).

Scott

**Comment by wilpig** [comment]
What is the poller doing exactly? If it's just hitting an API that seems like something that can be written in PHP quickly enough.

On May 2, 2017 6:00 AM, Scott Milliken <notifications@github.com> wrote:
Yes, that's fine to use Python for a support script (which is what I'd call
the poller script).

Scott

**Comment by samilliken** [comment]
It's likely just creating the API key for each request.  That's about the
only thing you really need the support libraries for.

Scott

**Comment by wilpig** [comment]
http://www.ovirt.org/develop/api/rest-api/rest-api/

Looking over their documentation it looks like it would be easy enough to convert whatever that python script is doing to straight api calls.

**Comment by wilpig** [comment]
I'm not against a helper script I am against adding additional language requirements that start to muddy the waters if it isn't necessary.

**Comment by wilpig** [comment]
And a simple search for ovirt php api turned up this...  https://github.com/layer7be/ovirt-php  I mean come on...  yes it says it requires php5.4 but without looking I am betting it is going to be more of the deferenced arrays being the crutch.

**Comment by ThomasADavis** [comment]
That PHP supports ovirt3x, and ovirt4x changed the rest API (and the SDK..)

I just do not have the time to convert/rewrite parts of their API into PHP; already have a python script that grabs this information, so it's just a matter of bolting on the mysql part.

For the first cut, start with the python script, then convert as time permits convert to PHP.
